eng/validation/update-channel.ps1

```diff
@@ -51,65 +51,35 @@ function Get-LatestBuildResult([PSObject]$repoData)
     return $true
 }
 
-$runtimeRepo = @{
-    azdoOrg = 'dnceng';
-    azdoProject = 'internal';
-    buildDefinitionId = 679;
-    githubRepoName = 'runtime';
-    subscribedBranchName = 'main'
-}
-$aspnetcoreRepo = @{
-    azdoOrg = 'dnceng';
-    azdoProject = 'internal';
-    buildDefinitionId = 21;
-    githubRepoName = 'aspnetcore';
-    subscribedBranchName = 'main'
-}
-$installerRepo = @{
-    azdoOrg = 'dnceng';
-    azdoProject = 'internal';
-    buildDefinitionId = 286;
-    githubRepoName = 'installer';
-    subscribedBranchName = 'main'
-}
-
-$bellwetherRepos = @($runtimeRepo, $aspnetcoreRepo, $installerRepo)
-
 $arcadeSdkPackageName = 'Microsoft.DotNet.Arcade.Sdk'
 $arcadeSdkVersion = $GlobalJson.'msbuild-sdks'.$arcadeSdkPackageName
 $getAssetsApiEndpoint = "$maestroEndpoint/api/assets?name=$arcadeSdkPackageName&version=$arcadeSdkVersion&api-version=$apiVersion"
 $headers = Get-Headers 'text/plain' $barToken
 
 try {
-    # Validate that the "bellwether" repos (runtime, installer, aspnetcore) are green on their main branches
-    $results = ($bellwetherRepos | ForEach-Object { Get-LatestBuildResult -repoData $_ })
```

Comment:
You are removing the usage for Get-LatestBuildResult, we should delete the function as well.

Reply:
lol, you beat me to it.. I just did :)